tests/snmpsim/fujitsueternusos.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.211.1.21.1.150
```

Comment:
My thought was that the sysDescr seems user defined in this case, hence not including it.
```
.1.3.6.1.2.1.1.1.0 = STRING: "RZ Storage"
```